addons/mass_mailing_sms/models/mailing_trace.py

```diff
@@ -65,6 +70,33 @@ def create(self, values_list):
                 values['sms_code'] = self._get_random_code()
         return super(MailingTrace, self).create(values_list)
 
+    @api.model
+    def fields_get(self, allfields=None, attributes=None):
+        # As we are adding keys in stable, better be sure no-one is getting crashes
+        # due to missing translations
+        # TODO: remove in master
+        res = super().fields_get(allfields=allfields, attributes=attributes)
+
+        existing_selection = res.get('failure_type', {}).get('selection')
+        if existing_selection is None:
+            return res
+
+        updated_stable = {
+            'twilio_authentication', 'twilio_callback',
+            'twilio_from_missing', 'twilio_from_to',
+        }
+        need_update = updated_stable - set(dict(self._fields['failure_type'].selection))
+        if need_update:
+            self.env['ir.model.fields'].invalidate_model(['selection_ids'])
+            self.env['ir.model.fields.selection']._update_selection(
+                self._name,
+                'failure_type',
+                self._fields['failure_type'].selection,
+            )
+            self.env.registry.clear_cache()
```

Comment:
Didn't we do that in `_init` for some other stuff, so it only runs at startup?

Reply:
doesn't `_init` only at install?

Reply:
Yes, same code lies in accounting ^^ . I agree it is a bit weird to do that at each fields_get call, but would like to be sure not to break all stable instances (well it requires to access twilio selection keys translations hence to use twilio), but it might happen sooner than expected, as it seems people are actually interested in that module.

Reply:
@flch-odoo Correcting myself here, indeed you're right `Model.init`/`Model._auto_init` is just for update/install
Not sure why I had in mind that constraints get updated without -u either actually..  

Anyway basically ignore this comment chain, there's no good way to do what I had in mind (i.e. run this at start-up, which I thought we did elsewhere)